external/fv3fit/tests/test_derived_model.py

```diff
@@ -20,11 +20,18 @@
 )
 
 
+def test_get_additional_inputs():
```

Comment:
Please add tests for dumping and loading DerivedModel. Specifically a test that if you dump and load and then predict, you get the same result as before you dumped.